framework/elements/BaseElement.js

```diff
@@ -107,9 +111,9 @@ class BaseElement {
 
   async getText() {
     const element = await this.getElement()
-    Logger.debug(`Getting the text from ${this.name}`)
+    Logger.trace(`Getting the text from ${this.name}`)
```

Comment:
please revert this change. it is really helpful to see the result of the method. 
I was talking about log level in wdio settings. if you wish you may set environment variable  and use its value to easily change log level instead of changing the code